test/resources/test_collection_exercise.py

```diff
@@ -0,0 +1,91 @@
+import json
+import unittest
+
+import requests_mock
+
+from ras_backstage import app
+
+
+url_get_survey_by_short_name = f'{app.config["RM_SURVEY_SERVICE"]}surveys/shortname/test'
+url_ces = '{}collectionexercises/survey/{}'.format(app.config["RM_COLLECTION_EXERCISE_SERVICE"],
+                                                   "cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87")
+url_ce = '{}collectionexercises/{}'.format(app.config["RM_COLLECTION_EXERCISE_SERVICE"],
```

Comment:
Just out of curiosity why have you used a mix of .format() and f for formatting urls here?

Reply:
I've done this because of line length, it might not be the best practice.

I prefer f-strings in general but I don't think they look better when you have to do them over multiple lines. So when I've been using .format it's usually because they went over the 100 character line length. 

Consistency might be better though so i could change